src/main/java/iudx/catalogue/server/authenticator/AuthenticationServiceImpl.java

```diff
@@ -46,8 +46,15 @@ static void validateAuthInfo(JsonObject authInfo) throws IllegalArgumentExceptio
         if (authInfo.isEmpty()) throw new IllegalArgumentException("AuthInfo argument is empty/missing");
         String token = authInfo.getString(TOKEN, "");
         String operation = authInfo.getString(OPERATION, "");
-        if (token.isBlank() || operation.isBlank())
+        if (token.isBlank() || operation.isBlank()) {
             throw new IllegalArgumentException("Token/Operation in authenticationInfo is blank/missing");
+        } else if (token.length() > TOKEN_SIZE) {
```

Comment:
Shall we make this check in api verticle level itself?

Reply:
As there was already a condition to check the empty/blank token, i thought best to add it here in AuthenticationServiceImpl.validateTokenInfo, it can also be added to the apiverticle (CrudApis) itself.

Reply:
Hmm. That is correct. It maybe wasteful. We could try using Java Closures to make it appear neater. 
Ok for now I will leave that as is.